app/pb_kits/playbook/index.js

```diff
@@ -1,3 +1,5 @@
+import "./_playbook.scss"
```

Comment:
This is needed for webpack to generate the css file, otherwise, it won't as it's not being used.